kolibri/core/assets/src/vue/loading-spinner/index.vue

```diff
@@ -10,37 +10,37 @@
 <script>
 
   module.exports = {
-
+    data: () => ({
+      timeoutId: undefined,
+      isVisible: false,
+    }),
+    methods: {
+      show() {
+        this.isVisible = true;
+      },
+    },
+    attached() {
+      this.isVisible = false;
+      this.timeoutId = window.setTimeout(this.show, 2500);
```

Comment:
As a small enhancement we could make this as a prop with a default value, but this if fine for now.